sdk/monitor/monitor-query/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,304 @@
+
+In case of single query, there can be only two possibilities for the value of `status` field of `result` object - `Success` or `PartialFailure`. You can access the details of the partially successful results by the following code snippet -
+
+```ts
+const result = await logsQueryClient.queryWorkspace(${azureLogAnalyticsWorkspaceId}, ${kustoQuery}, {
+    duration: ${duration}
+  });
+
+  if (result.status === LogsQueryResultStatus.Success) {
+    const tablesFromResult: LogsTable[] = result.tables;
+
+    if (tablesFromResult.length === 0) {
+      console.log(`No results for query '${kustoQuery}'`);
+      return;
+    }
+    console.log(`This query has returned table(s) - `);
+    processTables(tablesFromResult);
+  } else {
+    console.log(`Error processing the query '${kustoQuery}' - ${result.partialError}`);
+    if (result.partialTables.length > 0) {
+      console.log(`This query has also returned partial data in the following table(s) - `);
+      processTables(result.partialTables);
+    }
+  }
+```
+More details on the hierarchy of the response for single query can be found [here](https://github.com/KarishmaGhiya/azure-sdk-for-js/blob/monitor-query-TS/sdk/monitor/monitor-query/README.md#handle-logs-query-response)
+
+In case of multiple queries, there can be three possibilities for the value of `status` field of `result` object - `Success`, `PartialFailure` or `Failure`. You can access the details of the partially successful results by the following code snippet -
+
+```ts
+async function processBatchResult(result: LogsQueryBatchResult) {
+  let i = 0;
+  for (const response of result) {
+    console.log(`Results for query with query: ${queriesBatch[i]}`);
+    if (response.status === LogsQueryResultStatus.Success) {
+      console.log(
+        `Printing results from query '${queriesBatch[i].query}' for '${queriesBatch[i].timespan}'`
+      );
+      processTables(response.tables);
+    } else if (response.status === LogsQueryResultStatus.PartialFailure) {
+      console.log(
+        `Printing partial results from query '${queriesBatch[i].query}' for '${queriesBatch[i].timespan}'`
+      );
+      processTables(response.partialTables);
+      console.log(
+        ` Query had errors:${response.partialError.message} with code ${response.partialError.code}`
+      );
+    } else {
+      console.log(`Printing errors from query '${queriesBatch[i].query}'`);
+      console.log(` Query had errors:${response.message} with code ${response.code}`);
+    }
+    // next query
+    i++;
+  }
+}
+```
+More details on the hierarchy of the response for multiple queries can be found [here](https://github.com/KarishmaGhiya/azure-sdk-for-js/blob/monitor-query-TS/sdk/monitor/monitor-query/README.md#handle-logs-batch-query-response)
```

Comment:
Use the `Azure` org's link instead of your forked repo.
```suggestion
More details on the hierarchy of the response for multiple queries can be found [here](https://github.com/Azure/azure-sdk-for-js/blob/monitor-query-TS/sdk/monitor/monitor-query/README.md#handle-logs-batch-query-response)
```